src/test/resources/speedtest/travelSearch-expected-results-bdr.csv

```diff
@@ -1,3 +1,3 @@
 tcId,nTransfers,duration,cost,walkDistance,startTime,endTime,agencies,modes,routes,stops,details
-1,0,41m47s,-1,0,13:18:13,14:00:00,,,,,Unknown transit 0tx 41m47s
```

Comment:
Can you explain why this PR changes the non-C2 results?

Reply:
It is this fix(c7e20321). The minDurationRoutingStrategy now respect the alightSlack if no ConstrainedTransfers exist between to stops. This make the heuristic a bit tighter and the performance better. 

The code line is: 
https://github.com/opentripplanner/OpenTripPlanner/commit/c7e20321d5343bd9b522a524c7e9f6330495be5e#diff-13e08433be2742e1e92ec82e4ece37c45dc793afb140b5ec211bf1707bca1908R74

where we before used 0, and now pass in the slack.